docs/ae/releases-and-changelog/page-6.md

```diff
@@ -6,6 +6,13 @@ The changelog is the best place for in-depth overviews of what's new in Gravitee
 
 ## Alert Engine changelog
 
+### 2.1.5 (2023-08-18)
```

Comment:
Is it possible to change the general version of AE to 2.1.5 also ?

Reply:
@gravitee-io/tech-writers any suggestion here regarding changing the main version from 2.1.2 t o 2.1.5? I can trace `2.1.2` in couple of places [one of them is a docker command] and not sure where to update. Thanks

Reply:
done!